tests/smoke/test_movielens.py

```diff
@@ -50,53 +49,50 @@ def test_load_pandas_df(
     title_example,
     genres_example,
     year_example,
+    tmp_dir,

```

Comment:
there is no need to import this since it is a fixture that it is in conftest, see this: https://github.com/Microsoft/Recommenders/blob/master/tests/conftest.py#L4

Reply:
ohhhh good! will update

Reply:
ah, nvm. it means you don't need to do `import your_fixture_module`. You still need to pass the fixture object as an argument. I think the comments contain misleading contents.

Reply:
good point, yeah, what you said is more detailed. It would be nice to have it in the code